src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.c

```diff
@@ -36,16 +37,132 @@ static InitChainEntry D_80ABD750[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+Vec3f D_80ABD760[] = { 0.0f, 0.0f, 80.0f };
```

Comment:
This Vec3f doesn't seem to need to be an array, please change it back to a single element